core/src/test/java/org/infinispan/api/flags/FlagsEnabledTest.java

```diff
@@ -127,9 +128,19 @@ public void testReplicateSkipCacheLoad(Method m) {
       final String v = v(m, 1);
       final String k = k(m, 1);
       cache1.withFlags(Flag.SKIP_CACHE_LOAD).put(k, v);
-      // The write-skew check tries to load it from persistence.
-      assertLoadsAndReset(cache1, isTxCache() ? 1 : 0, cache2, 0);
 
+      Address primaryLocation = cache1.getDistributionManager().getPrimaryLocation(k);
```

Comment:
Here, I would use MagicKey, too. However, I am alerted by finding out several places where the 'production' code does not work and test do not show that as the entries are usually hashed in a 'lucky' way. This can be also source for some of the random test failures, if the cache start order is not 100% deterministic.
So pseudo-random hashing should be added to the list of ideas for improving the testsuite.